docs/sources/configuration/_index.md

```diff
@@ -334,6 +334,16 @@ The `frontend` block configures the Loki query-frontend.
 # CLI flag: -querier.max-outstanding-requests-per-tenant
 [max_outstanding_per_tenant: <int> | default = 100]
 
+# In the event a tenant is repeatedly sending queries that lead the querier to crash
+# or be killed due to an out-of-memory error, the crashed querier will be disconnected
+# from the query frontend and a new querier will be immediately assigned to the tenant’s shard.
+# This invalidates the assumption that shuffle sharding can be used to reduce the
+# impact on tenants. This option mitigates the impact by configuring a delay between when
+# a querier disconnects because of a crash and when the crashed querier is actually removed
+# from the tenant's shard.
+# CLI flag: -query-frontend.querier-forget-delay
+[querier_forget_delay: <duration> | default = 0s]
```

Comment:
Loki doesn't have support for querier shuffle sharding yet, I wonder if we should keep this configuration removed until it does?

Reply:
I don't see a problem leaving as is because its not only  related to shuffle sharding but I can remove it no problem

Reply:
@trevorwhitney any input here?

Reply:
I don't have as much context as @slim-bean, are you saying this value isn't even used by loki yet because tenants don't have shards? If so I can see the point that adding this field to the docs because changing it won't actually do anything. At the same time it is a setting we have that will appear when someone hits the `/config` endpoint or prints the config to stderr, so I don't think there's too much harm in documenting it to help explain how that value got there.